docs/containers.conf.5.md

```diff
@@ -710,11 +710,11 @@ and pods are visible.
 
 **network_cmd_path**=""
 
-Path to the slirp4netns binary.
+Path to the slirp4netns or pasta binary.
 
 **network_cmd_options**=[]
 
-Default options to pass to the slirp4netns binary.
+Default options to pass to the network binary.
```

Comment:
This is also not used for pasta
The option syntax is incompatible and because user might use both at the same time we cannot use the same field for slirp4netns and pasta.